build.sbt

```diff
@@ -8,7 +8,9 @@ version in ThisBuild := "3.4.0-SNAPSHOT"
 // The next release to be cut from master will be 3.4.0 during develop of 3.4.0 we check compatibility with 3.3.0.
 // The goal is not to stop any breaking change, but to make us aware. For each breaking change we should add MiMa exclusions.
 // This will also help us decide when a PR can be backported in 3.3.x branch.  
-binaryCompatSlickVersion in ThisBuild := Some("3.3.0") 
```

Comment:
Why was this necessary?

Reply:
Yeah, the comment tried to explain how to deal with MIMA warnings and this was not what was intended. The idea was that we could communicate the breaking changes if we saw all the excludes.